therapy/etl/chembl.py

```diff
@@ -220,42 +220,46 @@ def _load_alias(self, record, batch):
         record['aliases'] = record['aliases'].split("||")
 
         # Remove duplicates (case-insensitive)
-        record['aliases'] = \
-            list(set({a.casefold(): a for a in
+        aliases = \
+            list(set({t.casefold(): t for t in
                       record['aliases']}.values()))
 
-        if len(record['aliases']) > 20:
+        if len(aliases) > 20:
             del record['aliases']
         else:
-            for alias in record['aliases']:
+            for alias in aliases:
                 alias = {
                     'label_and_type': f"{alias.lower()}##alias",
                     'concept_id': f"{record['concept_id'].lower()}",
                     'src_name': SourceName.CHEMBL.value
                 }
                 batch.put_item(Item=alias)
+            # Remove case sensitive duplicates when loading therapy
+            record['aliases'] = list(set(record['aliases']))
```

Comment:
I think you could at least remove the comment on line 237, since it seems like it's more descriptive of what you're doing on 231-236 (we *do* want to keep differently-cased aliases when loading the identity item).

Also, you might a) be able to keep this as a set rather than a list, and b) might not need to make it a set at all, unless the chembl source DB has duplicate alias items (idk?)

Reply:
It should be a list since we want to return aliases as a list. 

ChEMBL might contain duplicates. I didn't do any checks when retrieving, so this is that check.